installbuilder/datafiles/Base_DSC.data

```diff
@@ -243,15 +231,31 @@ chown -R ${{RUN_AS_USER}} /opt/microsoft/omsagent/plugin
 # If this is ever changed/removed, coordinate with nxOMSAuditdPlugin author/maintainer(s)
 chown -R ${{RUN_AS_USER}} /opt/microsoft/omsconfig/Scripts
 
-# Set up the modules
+# Set up built-in resource modules for OMS DSC
 su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nx_1.0.zip 0"
 su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSPerfCounter_2.2.zip 0"
 su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSSyslog_2.2.zip 0"
 su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSKeyMgmt_1.0.zip 0"
 su - omsagent -c "/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxFileInventory_1.1.zip 0"
 
+# Set up a built-in resource module for DIY DSC that was removed during OMS Agent update
+# This is a temporary workaround to prevent DIY DSC from breaking after the OMS Agent is updated from version 1.4.3-225 or below
```

Comment:
>OMS Agent is updated from version 1.4.3-225 [](start = 76, length = 43)

which version is this ? i don't see it here [https://github.com/Microsoft/OMS-Agent-for-Linux/releases](https://github.com/Microsoft/OMS-Agent-for-Linux/releases). can we update with correct version ?

Reply:
Fixed